source/utilities.tex

```diff
@@ -854,12 +854,10 @@
 \end{itemdecl}
 
 \begin{itemdescr}
+Let \tcode{Vi} be \tcode{unwrap_ref_decay_t<Ti>}.
+
 \pnum
-\returns \tcode{pair<V1, V2>(std::forward<T1>(x), std::forward<T2>(y))},
-where \tcode{V1} and \tcode{V2} are determined as follows: Let \tcode{Ui} be
-\tcode{decay_t<Ti>} for each \tcode{Ti}. If \tcode{Ui} is a specialization
-of \tcode{reference_wrapper}, then \tcode{Vi} is \tcode{Ui::type\&},
-otherwise \tcode{Vi} is \tcode{Ui}.
+\returns \tcode{pair<V1, V2>(std::forward<T1>(x), std::forward<T2>(y))}.
```

Comment:
It would seem to simplify this spec if we use `unwrap_ref_decay_t<T1>` and `unwrap_ref_decay_t<T2>` directly here in the `\returns`, and drop both the previous "Let" paragraph and the following "where T_i is ..." paragraph.

Reply:
Done.